torch/csrc/jit/passes/quantization_patterns.h

```diff
@@ -63,8 +63,35 @@ graph(%packed_params, %a_quant, %r_scale, %r_zero_point, %r_dtype):
         %r = quantized::linear(%a_quant, %packed_params, %r_scale, %r_zero_point)
         return (%r) )";
 
+    std::string conv2d_relu = R"(
```

Comment:
I assume that this pattern is after inlining as I dont see module call methods here.

Reply:
yes, this is quant fusion pattern